infrastructure/src/main/java/org/corfudb/infrastructure/RemoteMonitoringService.java

```diff
@@ -717,9 +717,9 @@ private void correctWrongEpochs(PollReport pollReport) {
      *
      * @param layoutCompletableFutureMap Map of layout server endpoints to their layout requests.
      */
-    private void updateTrailingLayoutServers(
-            Map<String, CompletableFuture<Layout>> layoutCompletableFutureMap, Layout updatedLayout) {
+    private void updateTrailingLayoutServers(Map<String, CompletableFuture<Layout>> layoutCompletableFutureMap) {
 
+        final Layout latestLayout = serverContext.copyManagementLayout();
```

Comment:
add a comment so that we do not do change it by mistake.

Reply:
Done.